package.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "@nib-styles/header-small",
   "version": "0.0.1",
```

Comment:
Hmm actually removing the images is a breaking change if we're using this anywhere without react. @lauriejones can you think of anywhere we used it before react?

Can probably still get away with `0.1.0` cause of how `<1.0.0` versions are treated with `^` ranges.